src/rpcrawtransaction.cpp

```diff
@@ -166,7 +166,6 @@ void TxToJSON(const CTransaction& tx, const uint256 hashBlock, UniValue& entry)
 }
 
 UniValue getrawtransaction(const UniValue& params, bool fHelp)
-{   // ZEN_MOD_START
```

Comment:
don't remove bracket

Reply:
ACK, testing for more issues before pushing.